packages/block-editor/src/components/observe-typing/index.js

```diff
@@ -43,12 +43,62 @@ function isKeyDownEligibleForStartTyping( event ) {
 	return ! shiftKey && KEY_DOWN_ELIGIBLE_KEY_CODES.has( keyCode );
 }
 
+export function useMouseMoveTypingReset( ref ) {
```

Comment:
Would you mind adding docs explaining what this does especially since it's exported :)

Reply:
Of course, sorry about that. Added docs for `useTyingObserver` too.